### DISCUSSION TITLE
file name format

### DISCUSSION THREAD

**Discussion by btrem** [edent/SuperTinyIcons#781] [Q&A] [answered]
Most svg files are one word, even where the service is two or more words (e.g., redhat.svg, stackoverflow.svg, djangoproject.svg, etc.). But a few have an underscore to separate the words (e.g., linux_mint.svg, raspberry_pi.svg, internet_archive.svg, etc.). Is there a preferred format?

**Comment by edent** [comment, marked as answer]
It is very much whatever looks right at the time.
